src/routes/(admin)/dashboard/+page.svelte

```
@@ -1,11 +1,32 @@
+
+	$effect(() => {
+		const meaningfulParams = ['tag_ids', 'state_ids'];
+		const hasFilters = meaningfulParams.some((param) => {
+			const value = page.url.searchParams.get(param);
+			return value && value.trim() !== '';
+		});
+
+		const hasTagFilters = page.url.searchParams.getAll('tag_ids').length > 0;
+		const hasStateFilters = page.url.searchParams.getAll('state_ids').length > 0;
+
+		const result = !hasFilters && !hasTagFilters && !hasStateFilters;
```

ESLint

[error] 'result' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)